doc/book/admin/instance_config.rst

```diff
@@ -84,11 +82,51 @@ entire application business logic in it. We, however, do not recommend this,
 since it clutters the instance file and leads to unnecessary copy-paste when
 you need to run multiple instances of an application.
 
+.. _admin-tt-preload:
+
+Preloading Lua scripts and modules
```

Comment:
I would add the following information. Quoted the documentation request.

> Don't forget quotes, a shell interprets a semicolon on its own.
>
> If something goes wrong at script/module execution, tarantool reports
the problem and exits.
>
> A script receives a path to the file in `...`, a module gets the module
name. `arg` is the same as for the main script.

Reply:
Added these details.